cpp/include/cuml/fil/fil.h

```diff
@@ -71,6 +72,9 @@ enum output_t {
   /** output class label: either apply threshold to the output of the previous stage (for binary classification),
       or select the class with the most votes to get the class label (for multi-class classification).  */
   CLASS = 0x100,
+  SIGMOID_CLASS = SIGMOID | CLASS,
+  AVG_CLASS = AVG | CLASS,
+  AVG_SIGMOID_CLASS = AVG | SIGMOID | CLASS,
```

Comment:
it is easy to maintain this list of combos in header, and much less readable to do it outside the class, and keep `static const`ing the results

Reply:
This is fine.